srcpkgs/cjs/template

```diff
@@ -1,27 +1,29 @@
 # Template file for 'cjs'
 pkgname=cjs
-version=4.6.0
+version=4.8.0
 revision=1
-build_style=gnu-configure
+build_style=meson
 build_helper=gir
-configure_args="--disable-static --disable-profiler"
-hostmakedepends="automake autoconf-archive libtool gnome-common pkg-config
- gobject-introspection glib-devel dbus mozjs52-devel"
+configure_args="-Dskip_dbus_tests=true -Dskip_gtk_tests=true
+ -Dprofiler=disabled -Dinstalled_tests=false" # Disable tests because they need dbus, X and GTK, same as gjs.
```

Comment:
```suggestion
# XXX: disable tests because they need dbus, X and GTK, same as gjs
configure_args="-Dskip_dbus_tests=true -Dskip_gtk_tests=true
 -Dprofiler=disabled -Dinstalled_tests=false"
```

So someone else who touches the template knows this is something to be improved.

Reply:
Done, thank you!